src/PKSim.Presentation/Services/ProjectTask.cs

```diff
@@ -325,6 +329,14 @@ private void openProjectFromFile(string projectFile, bool shouldStartWorker = tr
          void openProject()
          {
             _workspace.OpenProject(projectFile);
+
+            if (_workspace.Project.MetaDataVersion < ProjectVersions.V11)
+            {
+               _workspace.Project.All<Individual>().Each(x =>
+               {
+                  _lazyLoadTask.Load(x);
```

Comment:
I don't undertand why we are not doing it when we load the project instead? Can we have a chat about this